src/components/sidebarStorageTable/StorageTable.tsx

```diff
@@ -0,0 +1,29 @@
+// import { FC } from 'react'
+import React from "react";
+import Document from '../../interfaces/storage.interface'
+import { ItemListStorage } from '../ItemListStorage/ItemListStorage'
+
+import './StorageTable.scss'
+
+export interface DocumentProps {
+    documents: Document[];
+}
+export const StorageTable: React.FC<DocumentProps> = ({ documents }: DocumentProps) => {
+    return (<div className="storage__table-data">
+        <table>
+            <tbody>
+                {documents.map(({ id, document, file, storage, color }) => (
+                    <ItemListStorage
+                        id={id}
```

Comment:
It's important to always include the `key` element when generating an array of react components
```suggestion
                        id={id}
                        key={id}
```

Reply:
If you're right, it happened to me 😅